impl/src/main/java/com/okta/sdk/impl/oauth2/AccessTokenRetrieverServiceImpl.java

```diff
@@ -132,16 +132,16 @@ String createSignedJWT() throws InvalidKeyException, IOException {
         Instant now = Instant.now();
 
         JwtBuilder builder = Jwts.builder()
-            .setAudience(tokenClientConfiguration.getBaseUrl() + TOKEN_URI)
-            .setIssuedAt(Date.from(now))
-            .setExpiration(Date.from(now.plus(50, ChronoUnit.MINUTES)))             // see Javadoc
-            .setIssuer(clientId)
-            .setSubject(clientId)
+            .audience().add(tokenClientConfiguration.getBaseUrl() + TOKEN_URI).and()
```

Comment:
The openapi code generator changed your method signatures? Do you have a rev a major or minor version of okta-sdk-java now that all the methods have changed?

Reply:
No, this change is due to the bump in jjwt library - https://github.com/jwtk/jjwt/releases/tag/0.12.5